src/api/app/lib/backend/connection.rb

```diff
@@ -22,13 +24,52 @@ class Connection
       end
     end
 
+    def self.ssl_options()
+      unless @@ssl_options
```

Comment:
It'll probably be better to do this in an initializer once and store this in a [class accessor](https://api.rubyonrails.org/classes/Module.html#method-i-mattr_accessor). so this is per rails process (which we have many of) instead of per instantiated class (we do this often just for one call).